addons/hr_payroll/hr_payroll.py

```diff
@@ -404,7 +404,7 @@ def was_on_leave(employee_id, datetime_day, context=None):
             day_to = datetime.strptime(date_to,"%Y-%m-%d")
             nb_of_days = (day_to - day_from).days + 1
             for day in range(0, nb_of_days):
-                working_hours_on_day = self.pool.get('resource.calendar').working_hours_on_day(cr, uid, contract.working_hours, day_from + timedelta(days=day), context)
+                working_hours_on_day = self.pool.get('resource.calendar').working_hours_on_day(cr, uid, contract.working_hours, day_from + timedelta(days=day), context=context)
```

Comment:
@atp-odoo  I'm not facing any trouble without this change. When will we need this?